File: user_server/models/Log.go
package models

type Log struct {
	ID          int    `json:"id" gorm:"primaryKey, autoIncrement"`
	AppName     string `json:"app_name"`                          
	LogType     string `json:"log_type"`                     
	Module      string `json:"module"`                            
	LogDateTime string `json:"log_date_time"`                  
	Summary     string `json:"summary"`                         
	Description string `json:"description"`                        
}

File: health_server/models/log.js

class Log {
    constructor(ID, app_name, log_type, module, log_date_time, summary, description) {
      this.ID = ID;
      this.app_name = app_name;
      this.log_type = log_type;
      this.module = module;
      this.log_date_time = log_date_time;
      this.summary = summary;
      this.description = description;
    }
  }  

module.exports = Log;
